lib/polarshift/test_record.rb

```diff
@@ -173,9 +173,11 @@ def update_to!(new_state)
       def start_scenario_for!(test_case)
         scenario = self.test_case.scenarios.find { |s| s.match! test_case}
         unless scenario
-          raise "logic error: trying to start Cucumber scenario not part of this test record"
+          puts "logic error: trying to start Cucumber scenario not part of this test record\n"
```

Comment:
Shall we use `logger.warn` instead of `puts`?

Reply:
done...